packages/core/src/components/datetime/test/success/datetime.e2e.ts

```diff
@@ -0,0 +1,31 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/datetime/test/success/index.html';
+
+test.describe('tds-datetime-success', () => {
+  test('renders success datetime component correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    /* Check diff on screenshot */
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('component should have size "md"', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const dateTime = page.locator('tds-datetime');
+    // Check for the label text
+    const label = page.locator('.tds-datetime-label');
+    await expect(label).toBeVisible();
+    await expect(label).toHaveText('Label text');
+
+    // Check for the helper text
+    const helperText = page.locator('.tds-datetime-helper .tds-helper');
+    await expect(helperText).toBeVisible();
+    await expect(helperText).toHaveText('Helper text');
+
+    // For example, checking for the "state" attribute to be "success"
+    await expect(dateTime).toHaveAttribute('state', 'success');
+    await expect(dateTime).toHaveAttribute('size', 'md');
```

Comment:
Here maybe it would be good to check some CSS to confirm the success state and correct height for medium.

Reply:
Spreasheet specifies checking more attributes and doing selections and checking today's date that are not part of unit test here. Was leaving them out intentional?

Reply:
There are some cases that are related to interactions with the datepicker we open, but as it is a native one there are some limitations to what we can do in unit tests. If that is what you mean?